src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/CHANGELOG.md

```diff
@@ -2,23 +2,22 @@
 
 ## Unreleased
 
-* Updated the `ActivitySource` name to the assembly name:
-  `OpenTelemetry.Instrumentation.AWSLambda`
+## 2.0.0
+
+* BREAKING (behavior): Update the `ActivitySource` name to not include `Contrib`.
+  The new activity source name is `OpenTelemetry.Instrumentation.AWSLambda`.
```

Comment:
This rewording should fix https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/534#discussion_r947204413

I hope the activity source name actually conforms to the current guidelines? If not, I would like to revert #534 before the release naturally.

Reply:
Yes, the ActivitySource name matches the standard practice.